airflow/www/app.py

```diff
@@ -109,6 +109,11 @@ def create_app(config=None, testing=False):
         av(vs.DagRunModelView(
             models.DagRun, Session, name="DAG Runs", category="Browse"))
         av(vs.DagModelView(models.DagModel, Session, name=None))
+
+        http_test = vs.TestAPI.as_view('http_test')
```

Comment:
Why do it this way? http_test is not a normal endpoint and it should not be there in production.